examples/model_compress/amc/mobilenet_v2.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
do you think it is better to put these mobilenet models into the model folder?

Reply:
Thanks, model fold looks better, will move it into model folder.